### PR TITLE
Release 1.0 prep work

### PR DESCRIPTION
**Pull request by woodruffw**
Prep work for a 1.0 release.

### CODE REVIEW

**Comment by woodruffw** [comment]
Key changes:

1. We no longer support Python 2 builds of pepy.
2. pe-parse is now versioned according to the top-level `VERSION` file.
3. The version in `VERSION` is propagated to pe-parse as `PEPARSE_VERSION`, which pepy in turn propagates as `PEPY_VERSION` and `__version__` as module-level constants (in addition to `PEPARSE_VERSION`).

**Comment by ekilmer** [comment]
Could you add a link to the `pepy` README in the top-level README? Maybe at the end of the `Building` section?

**Comment by woodruffw** [comment]
> Could you add a link to the `pepy` README in the top-level README? Maybe at the end of the `Building` section?

Definitely! I didn't actually realize there wasn't one in there 😅

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -32,18 +32,65 @@ jobs:
         cd build
         cmake ..
         make
```
Comment (ekilmer) [inline .github/workflows/ci.yml]: Should we change the CMake build commands to be the same as what's in the README?

Comment (woodruffw) [reply .github/workflows/ci.yml]: Yeah, good idea. I'll also parametrize the build type.

Comment (woodruffw) [reply .github/workflows/ci.yml]: (I don't believe the `--config Release` args are necessary after the configure step, so I'll remove those from the README).

**Comment by woodruffw** [comment]
> Or am I missing something?

Installing the sdist from PyPI _should_ work on Windows (since it's a direct copy of the `setup.py` and all extension files). I think the problem in the CI is with installing the sdist from a local file, since it's going through PowerShell and probably is getting confused by the glob I passed it. I didn't want to spent too much time debugging it since the sdist we publish to PyPI will be authoritative in any case.

**Comment by ekilmer** [comment]
> it's going through PowerShell and probably is getting confused by the glob I passed it. I didn't want to spent too much time debugging it since the sdist we publish to PyPI will be authoritative in any case.

Ah okay. Makes sense. Looks good then 👍

**Pull request merged**